cmd/utils.go

```diff
@@ -636,6 +638,26 @@ func getGitLabels(dryrun bool) (map[string]string, error) {
 		}
 	}
 
+	if commitInfo.Author != "" && commitInfo.AuthorEmail != "" {
```

Comment:
I'm not sure if git allows one of these without the other, but I think we should still add the label if it is just the name or the email. If both are empty strings, skip the label. (same goes for committer)

Reply:
Apparently the name is always there: https://stackoverflow.com/questions/11579311/git-commit-as-different-user-without-email-or-only-email. So will change to allow for only the name, and then append the email if it exists.

Reply:
That sounds good.
Looks like you forgot to remove the `&& commitInfo.AuthorEmail != ""` from this if check